Sources/Support/DebugUI/DebugContentViews.swift

```diff
@@ -12,6 +12,7 @@
 //  Created by Nacho Soto on 5/30/23.
 
 #if DEBUG && swift(>=5.8) && (os(iOS) || os(macOS))
+#if !(swift(>=5.9) && os(xrOS))
```

Comment:
Hmm can it be `xrOS` after the previous line (`(os(iOS) || os(macOS)`)? Same in the other places in this file

Reply:
I think `os(xrOS)` is only available in `swift(>=5.9)` so that would mean this code would not compile in Xcode 14 (@NachoSoto correct me if I'm wrong)